extension/js/common/platform/xss.ts

```diff
@@ -47,6 +54,22 @@ export class Xss {
     });
   }
 
+  /**
+   * Sanitize HTML to protect from nasty user-generated external content, and make it "less rich"
+   *
+   * This method is less tolerant and sanitizes everything we don't belive should be sent as email content.
+   * Used when rendering rich-text emails (from untrusted sources - that is, all sources).
+   *
+   * @property {string} dirtyHtml     - HTML to be sanitized
```

Comment:
I guesst it should be `@param`, not `@property`

Reply:
You're right. I haven't used these in a while.